### DISCUSSION TITLE
Unable to prevent scroll using stop()

### DISCUSSION THREAD

**Discussion by pulpmint** [darkroomengineering/lenis#361] [Q&A]
I'm trying to pause scrolling when a pop-up / menu component is toggled by using `stop()` but it is not working.

I can see that the class `lenis-stopped` is added to the `<html>` tag but whenever I'm scrolling, `lenis-scrolling` is also getting added & I'm still able to scroll.

I've also added the styes in the `global.css` mentioned in the document & I also tried marking `overflow: hidden` as `!important` but it did not work.

**Comment by clementroche** [comment]
please share a reproduction link @pulpmint

**Comment by pulpmint** [comment]
NM: Found the issue. There was something wrong with the way I was importing the component (where I was initialising Lenis) at the page level. Closing this.
